playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -120,7 +120,72 @@
     set_fact:
       r_etcd_migrate_success: true
 
-- include: ./scaleup.yml
+- name: Gather facts for migrated nodes
+  hosts: oo_etcd_to_config:oo_new_etcd_to_config
+  roles:
+  - etcd_common
+  - openshift_etcd_facts
+  post_tasks:
+  - set_fact:
+      etcd_hostname: "{{ etcd_hostname }}"
+      etcd_ip: "{{ etcd_ip }}"
+      etcd_url_scheme: "{{ etcd_url_scheme }}"
+      etcd_peer_url_scheme: "{{ etcd_peer_url_scheme }}"
+      etcd_peer_port: "{{ etcd_peer_port }}"
+      etcd_client_port: "{{ etcd_client_port }}"
+      etcd_conf_file: "{{ etcd_conf_file }}"
+      etcd_peer_cert_file: "{{ etcd_peer_cert_file }}"
+      etcd_peer_key_file: "{{ etcd_peer_key_file }}"
+      etcd_peer_ca_file: "{{ etcd_peer_ca_file }}"
+
+
+- name: Re-configure etcd and bring the cluster up
+  hosts: oo_new_etcd_to_config
+  serial: 1
+  any_errors_fatal: true
+  vars:
+    etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
+  tasks:
+  - name: Add new etcd members to cluster
+    command: >
+      /usr/bin/etcdctl --cert-file {{ etcd_peer_cert_file }}
+                       --key-file {{ etcd_peer_key_file }}
+                       --ca-file {{ etcd_peer_ca_file }}
+                       -C {{ etcd_peer_url_scheme }}://{{ hostvars[etcd_ca_host].etcd_hostname }}:{{ etcd_client_port }}
+                       member add {{ etcd_hostname }} {{ etcd_peer_url_scheme }}://{{ etcd_ip }}:{{ etcd_peer_port }}
+    delegate_to: "{{ etcd_ca_host }}"
+    failed_when:
+    - etcd_add_check.rc == 1
+    - ("peerURL exists" not in etcd_add_check.stderr)
+    register: etcd_add_check
+    retries: 3
+    delay: 10
+    until: etcd_add_check.rc == 0
+  - name: Set ETCD_INITIAL_CLUSTER_STATE=existing on migrated etcd host
+    lineinfile:
+      line: "ETCD_INITIAL_CLUSTER_STATE=existing"
```

Comment:
Should we add regex to this step?  Don't want to define this variable twice.

Reply:
Good idea, implemented